tesseract_collision/include/tesseract_collision/core/common.h

```diff
@@ -34,7 +34,9 @@ TESSERACT_COLLISION_IGNORE_WARNINGS_PUSH
 #include <Eigen/Geometry>
 #include <fstream>
 #include <iostream>
+#include <iomanip>
```

Comment:
Is this needed?

Reply:
I don't think so. I will remove it and see if things compile.